### ISSUE TITLE
Closed status need a new post 

### ISSUE THREAD

**Issue by AlessandroMelzi** [pinkynrg/convergence2.0#171]
Closed status should be a process after Solved status and should not need a new post to change the status.

**Comment by pinkynrg** [comment]
tonight I will write a routine the closes solved tickets that have been solved longer than 2 months. In case you want to close them before you'll need to write a post. I would like any user action to be explained with a post.

**Comment by pinkynrg** [comment]
Why do you think a post is not necessary?
 my idea here is more info a better then less info.

**Comment by AlessandroMelzi** [comment]
Because the change from Solved to Closed is used only when the customer is not requesting to open again the ticket.

**Comment by e80tom** [comment]
You know that in general I don't use the closed status, but I know ticketing systems use them, so it may be useful in the future.

We need to ask, what does the close closed status do for us. It tempts our ability to change it, but it also does for the customer, so maybe it'll be some pressure for them to act before the issue is closed for good. 

I think we can keep the post for closed. We may want to think about how we use our statuses more. We can use solved a bit more like we use waiting feedback after we put a change along with a message that tells the customer it'll be closed after a certain amount of days, and also a reminder when it gets closer to the close date. That'll induce action.

In any case, we want the root cause analysis to be the post at the top, rather than a bunch of tickets that say, automatically closed, or customer agrees this issue is closed.

**Comment by e80tom** [comment]
...Prevents\* out ability...

**Comment by pinkynrg** [comment]
I agree. That in fact, would not be only a bunch of ticket closing but a
motivational emails to either invite the customer to follow up or remark
one final time before closing,  what for us, is considered a solved and
archivable issue.

**Comment by AlessandroMelzi** [comment]
I agree about the automatically closed and the notification to the customer. 
For the root cause I think is very important to have this post on top of all the other posts. The root cause should be written on the solved status or before that.

**Comment by pinkynrg** [comment]
the idea about the "post at the top" is whenever the ticket is WCF, solved or closed we have a post at the top describing why/how the ticket has been changed to that status
